source/MRVoxels/MRVDBConversions.h

```diff
@@ -58,6 +58,10 @@ MRVOXELS_API VdbVolume floatGridToVdbVolume( FloatGrid grid );
 // grid can be used to make iso-surface later with gridToMesh function
 MRVOXELS_API FloatGrid simpleVolumeToDenseGrid( const SimpleVolume& simpleVolume, ProgressCallback cb = {} );
 MRVOXELS_API VdbVolume simpleVolumeToVdbVolume( const SimpleVolumeMinMax& simpleVolume, ProgressCallback cb = {} );
+MRVOXELS_API void simpleVolumeToDenseGrid(
```

Comment:
please add specific comment to this function, how to use it. Probably give another name to it?